drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -79,6 +79,14 @@ static const clock_ip_name_t sai_clocks[] = {
 	kCLOCK_AUDIO_Sai3,
 };
 #endif
+
+#if defined(CONFIG_SOC_MIMX8ML8_ADSP)
+static const clock_ip_name_t sai_clocks[] = {
+	kCLOCK_Sai1,
+	kCLOCK_Sai2,
+	kCLOCK_Sai3,
+};
+#endif /* CONFIG_SOC_MIMX8ML8_ADSP */
 #endif /* CONFIG_DAI_NXP_SAI */
```

Comment:
There is a macro `SAI_CLOCKS` for this in the HAL, but I think it is clearer the way you did it.